cop_prinstrat/Makefile

```diff
@@ -0,0 +1,39 @@
+## all           : clean saved results and get results to populate report
+all: clean cop_prinstrat 
+
+## clean         : delete saved bootstrap results
+clean:
+	rm -rf output/*
+
+## cop_prinstrat     : create tables and figures
+cop_prinstrat:
+ifeq ($(TRIAL), moderna_mock)
```

Comment:
I don't think this will work because it does not have a `tab`. Have you confirmed whether this works? It may need to be modified to something like 

```bash
cop_prinstrat:
    ifeq($(TRIAL), moderna_mock) \
        ... \
        ... \
```